lib/manager/components/CreateFeedSource.js

```diff
@@ -93,6 +99,32 @@ export default class CreateFeedSource extends Component<Props, State> {
     }
   )
 
+  _intervalsForFreq = (fetchFrequency: FetchFrequency) => {
```

Comment:
This is reused in `GeneralSettings`. Perhaps it ought to be a util function so it can be reused.